sdk/communication/Azure.ResourceManager.Communication/tests/ScenarioTests/SuppressionListTests.cs

```diff
@@ -15,266 +15,266 @@
-            }
-
-            await suppressionList.DeleteAsync(WaitUntil.Started);
-        }
-
-        [Test]
-        public async Task Update_SuppresionListAddress()
-        {
-            var listName = Recording.Random.NewGuid().ToString();
-            var suppressionList = await CreateDefaultSuppressionListResource(_domainResource, listName);
-
-            Assert.IsNotNull(suppressionList);
-
-            var email = "superuser@email.com";
-            var created = await CreateDefaultSuppressionListAddressResource(suppressionList, email, "firstName", "lastName", "notes");
-
-            SuppressionListAddressResourceData data = new SuppressionListAddressResourceData
-            {
-                Email = created.Data.Email,
-                FirstName = $"updated+{created.Data.FirstName}",
-                LastName = $"updated+{created.Data.LastName}",
-                Notes = created.Data.Notes
-            };
-
-            var updated = await suppressionList
-                .GetSuppressionListAddressResources()
-                .CreateOrUpdateAsync(WaitUntil.Completed, created.Id.Name, data);
-
-            Assert.IsNotNull(updated);
-            Assert.AreEqual(created.Data.Id, updated.Value.Data.Id);
-            Assert.AreEqual(created.Data.Email, updated.Value.Data.Email);
-            Assert.AreNotEqual(created.Data.FirstName, updated.Value.Data.FirstName);
-            Assert.AreNotEqual(created.Data.LastName, updated.Value.Data.LastName);
-            Assert.AreEqual(created.Data.Notes, updated.Value.Data.Notes);
-
-            await created.DeleteAsync(WaitUntil.Started);
-            await suppressionList.DeleteAsync(WaitUntil.Started);
-        }
-
-        [Test]
-        public async Task Delete_SuppresionListAddress()
-        {
-            var listName = Recording.Random.NewGuid().ToString();
-            var suppressionList = await CreateDefaultSuppressionListResource(_domainResource, listName);
-
-            Assert.IsNotNull(suppressionList);
-
-            var email = "superuser@email.com";
-            var addressResource = await CreateDefaultSuppressionListAddressResource(suppressionList, email, "firstName", "lastName", "notes");
-
-            var collection = suppressionList.GetSuppressionListAddressResources();
-
-            var exists = await collection.ExistsAsync(addressResource.Id.Name);
-            Assert.IsTrue(exists);
-
-            await addressResource.DeleteAsync(WaitUntil.Completed);
-
-            // // todo: follow up on this issue. getting item after being deleted should return 404 not found instead of 500.
-            // collection = suppressionList.GetSuppressionListAddressResources();
-            // exists = await collection.ExistsAsync(addressResource.Id.Name);
-            // Assert.IsFalse(exists);
-
-            await suppressionList.DeleteAsync(WaitUntil.Started);
-        }
-    }
+    //public class SuppressionListTests : CommunicationManagementClientLiveTestBase
```

Comment:
why is the whole file commented out?

Reply:
They are scenario tests written for a preview SDK off this API version. The tests are still relevant, not just for this Version.